Lidgren.Network/NetQueue.cs

```diff
@@ -326,12 +328,12 @@ public bool Contains(T item)
 		/// <summary>
 		/// Copies the queue items to a new array
 		/// </summary>
-		public T[] ToArray()
+		public T?[] ToArray()
 		{
 			m_lock.EnterReadLock();
 			try
 			{
-				T[] retval = new T[m_size];
+				T?[] retval = new T[m_size];

```

Comment:
It shouldn't be possible for the queue to contain null entries within the actual valid range, so these shouldn't be nullable.